index.js

```diff
@@ -7,12 +10,28 @@ function ask(questionText) {
   });
 }
 
+//----------------async code---------------//
+
 start();
 
 async function start() {
+  let max = 100
+  let min = 1
```

Comment:
If you look back into the Playing with Numbers labs, you will find the expression for generating a random number which you may want to make the value of `compNumber`